src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+
+const sort = (target) => {
+  let index = null;
+
+  switch (target) {
+    case 'Name':
+      index = 0;
+      break;
+
+    case 'Position':
+      index = 1;
+      break;
+
+    case 'Age':
+      index = 2;
+      break;
+
+    case 'Salary':
+      index = 3;
+      break;
+  }
```

Comment:
remove index at all, write all logic inside cases (you can combine them as well if you need to do this)